### PR TITLE
Add payload checks on E2E login and cluster HTTP requests tests

### LINKED ISSUE #7275: Print payloads and response from E2E in CI
# Description

Create [`cy.interceptor()`](https://docs.cypress.io/api/commands/intercept) command for any HTTP request in Cypress to print out with [`cy.log()`](https://docs.cypress.io/api/commands/log) both payload and response in case of errors.

## Context

In some cases, like for the [422 status at login time](https://github.com/rancher/dashboard/actions/runs/3299019525/jobs/5467098313#step:6:264), it is not possible to identify the reason of this response. More informations are required to be returned in these cases for debugging purposes.

Note: This is a temporary solution as no error monitoring tool is available in the current state. It is recommended to avoid public print out for multiple reasons, on top of convenience of a proper dedicated tool for such scope.

### PR DESCRIPTION
**Pull request by cnotv**
### Summary
Fixes #7275

### Occurred changes and/or fixed issues
Added check for login command and cluster tests.

### Technical notes summary
- Please refer to issues notes for current state of actions
- Updated Cypress to `10.11.0`
- Enabled Cypress Studio as playground
- Fixed issues after Cypress upgrade:
  - Removed waiting time for event triggered by CTAs
  - ~Added waiting time for refresh request after deleting cluster~ Reloaded page after delete, due unclear side effects
- Restricted test retry to 2 for run mode only, due issues with the snapshots in the Cypress Dashboard
- Allowed to change Cypress waiting time from env var

### Areas or cases that should be tested
Checks should behave in the same way.

### Areas which could experience regressions
Test workload.

### Screenshot/Video
Please refer checks below.

### CODE REVIEW

**Comment by cnotv** [comment]
PR fails after merge to master due to architecture changes.

File: cypress/e2e/tests/pages/cluster-manager.spec.ts [hunk 1]
```diff
@@ -35,6 +41,15 @@ describe('Cluster Manager', () => {
       const editCreatedClusterPage = new ClusterManagerEditRke2CustomPagePo(rke2CustomName);
 
       it('can create new cluster', () => {
+        cy.intercept('POST', `/v1/${ type }s`).as('createRequest');
+        const request = {
+          type,
+          metadata: {
+            namespace,
+            name: clusterName
```
Comment (richard-cox) [inline cypress/e2e/tests/pages/cluster-manager.spec.ts:49]: This should be `rke2CustomName`

File: cypress/e2e/tests/pages/cluster-manager.spec.ts [hunk 2]
```diff
@@ -114,6 +136,17 @@ describe('Cluster Manager', () => {
         importClusterPage.create();
 
         detailClusterPage.waitForPage(undefined, 'registration');
+
+        cy.wait('@importRequest').then((intercept) => {
+          expect(intercept.request.body).to.deep.equal({
+            type,
+            metadata: {
+              namespace,
+              name: clusterNameImport
```
Comment (richard-cox) [inline cypress/e2e/tests/pages/cluster-manager.spec.ts:145]: This should be `importGenericName`

File: cypress/e2e/tests/pages/cluster-manager.spec.ts [hunk 3]
```diff
@@ -14,10 +16,14 @@ const runPrefix = `e2e-test-${ runTimestamp }`;
...
 // File specific consts
 const { baseUrl } = Cypress.config();
-const clusterRequestBase = `${ baseUrl }/v1/provisioning.cattle.io.clusters/fleet-default`;
+const namespace = 'fleet-default';
+const type = 'provisioning.cattle.io.cluster';
+const clusterRequestBase = `${ baseUrl }/v1/${ type }/${ namespace }`;
 const clusterNamePartial = `${ runPrefix }-create`;
 const rke2CustomName = `${ clusterNamePartial }-rke2-custom`;
 const importGenericName = `${ clusterNamePartial }-import-generic`;
+const clusterNameImport = `${ clusterNamePartial }-${ runTimestamp }-import`;
```
Comment (richard-cox) [inline cypress/e2e/tests/pages/cluster-manager.spec.ts:25]: With the changes below both `clusterNameImport` and `clusterName` can be removed

**Review by richard-cox — CHANGES_REQUESTED** [review]

**Comment by cnotv** [comment]
@richard-cox it still fails, still no idea why.

File: cypress/e2e/tests/pages/cluster-manager.spec.ts [hunk 4]
```diff
@@ -14,7 +16,9 @@ const runPrefix = `e2e-test-${ runTimestamp }`;
 
 // File specific consts
 const { baseUrl } = Cypress.config();
-const clusterRequestBase = `${ baseUrl }/v1/provisioning.cattle.io.clusters/fleet-default`;
+const namespace = 'fleet-default';
+const type = 'provisioning.cattle.io.cluster';
```
Comment (richard-cox) [inline cypress/e2e/tests/pages/cluster-manager.spec.ts:20]: There's a diff between the new and old (`s`)

`provisioning.cattle.io.clusters` vs `provisioning.cattle.io.cluster`

**Review by richard-cox — CHANGES_REQUESTED** [review]

File: cypress/e2e/tests/pages/cluster-manager.spec.ts (hunk 4 above)
Comment (cnotv) [reply cypress/e2e/tests/pages/cluster-manager.spec.ts:20]: nice catch, thanks :)

**Comment by cnotv** [comment]
> @cnotv think this one needs a rebase, and then is ready to review & go in?

Gonna give it a check after all this time.

**Review by richard-cox — APPROVED** [review]

**Pull request merged**